node-yun-http/platformio.ini

```diff
@@ -0,0 +1,27 @@
+; PlatformIO Project Configuration File for Mois sensor node
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/en/latest/projectconf/
+
+[platformio]
+src_dir = .
+
+monitor_speed = 115200
+
+[common]
+lib_deps =
+    ../libraries/ADS1231
```

Comment:
This is the only library still pulled in from what we provide using Git submodules. On the other hand, I see that you put up a repository at [^1]. I think it will make sense to also provide the original ADS1231 library by @jeffkub using a dedicated repository. I think it is long overdue, do you agree?

[^1]: https://github.com/bee-mois/ads1234

Reply:
Oh, @hardmax might have done that already. That looks promising, thank you so much!

-- https://github.com/hardmax/ADS1232
-- https://registry.platformio.org/libraries/hardmax/ADS1232

Reply:
I've diverted that change to GH-54. Not sure if it will work at all. Please respond over there.